src/Resources/views/standard_layout.html.twig

```diff
@@ -157,7 +158,8 @@ file that was distributed with this source code.
                                                 {% if action is defined %}
                                                     {% for menu in breadcrumbs_builder.breadcrumbs(admin, action) %}
                                                         {%- set translation_domain = menu.extra('translation_domain', 'messages') -%}
-                                                        {%- set label = menu.label -%}
+                                                        # We use method accessor instead of ".label" since `menu` implements `ArrayAccess` and could have a property called "label".
```

Comment:
```suggestion
                                                        {# We use method accessor instead of ".label" since `menu` implements `ArrayAccess` and could have a property called "label". #}
```